src/community/label_propagation.jl

```diff
@@ -0,0 +1,99 @@
+"""
+Community detection using the label propagation algorithm (see [Raghavan et al.](http://arxiv.org/abs/0709.2938)).
+`g`: imput Graph
+`maxiter`: optional, maximum iterations
+return : array containing vertex assignments, the range of the output as 1:countdistinct(membership)
+"""
+function label_propagation(g::SimpleGraph; maxiter=1000)
+    n = nv(g)
+    membership = collect(1:n)
+    active_nodes = Set(vertices(g))
```

Comment:
It might be more efficient to use an `IntSet` here, especially for larger graphs - could you try that?